homeassistant/components/elmax/common.py

```diff
@@ -168,21 +182,19 @@ def name(self) -> str | None:
         """Return the entity name."""
         return self._device.name
 
-    @property
-    def extra_state_attributes(self) -> Mapping[str, Any] | None:
-        """Return extra attributes."""
-        return {
-            "index": self._device.index,
-            "visible": self._device.visible,
-        }
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self._transitory_state = None
+        self.async_write_ha_state()
```

Comment:
Why do we need this? The coordinator & coordinator entity already handle this?

Reply:
The standard coordinator update method only triggers `self.async_write_ha_state()` bue does not reset my _transitory_state variable. Now that I think of it, it would probably be better to invoke the super implementation after I reset my transitory state variable... isn't it?

```python
    def _handle_coordinator_update(self) -> None:
        """Handle updated data from the coordinator."""
        self._transitory_state = None
        super._handle_coordinator_update()
```

Reply:
>  bue does not reset my _transitory_state variable.

You don't need `_transitory_state` at all, that is my whole point.

Reply:
Resolving this thread as the same discussion is being addressed into another place.